homeassistant/components/wemo.py

```diff
@@ -61,9 +61,8 @@ def stop_wemo(event):
 
     def discovery_dispatch(service, discovery_info):
         """Dispatcher for WeMo discovery events."""
-        # name, model, location, mac
-        model_name = discovery_info.get('model_name')
-        serial = discovery_info.get('serial')
+        model_name = discovery_info[1]
```

Comment:
So I can't run this locally after the regression (even after re-pulling dev, rebasing and re-running `script/setup`) - so I must be picking up an incorrect version of netdisco.

````
17-04-17 10:55:30 ERROR (Thread-4) [homeassistant.core] Error doing job: Future exception was never retrieved
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.5.2_3/Frameworks/Python.framework/Versions/3.5/lib/python3.5/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/greg/ha/home-assistant/homeassistant/components/wemo.py", line 65, in discovery_dispatch
    model_name = discovery_info.get('model_name')
AttributeError: 'tuple' object has no attribute 'get'
````

The rest of the PR should be fine.